docs/packaging/updating-an-existing-package.md

```diff
@@ -167,6 +167,21 @@ Full release notes:
 - [ ] Package was built and tested against unstable
 ```
 
+### Other commit message format examples
+
+In the cases where you are not updating a package to a new version, but simply applying a patch or fixing an existing version, the following commit message shortlogs are considered suitable:
+
+- Add the `homepage` metadata key:
+  - `package: Add homepage`
+- Fix a packaging issue (including adding a patch to fix the package)
+  - `package: Fix (...)`
+- Updating a README, monitoring.yml file or other stuff that does not directly _change_ the package, meaning the package release number didn't need a bump:
+  - `package: [NFC] Add README.md and monitoring.yml`
```

Comment:
I don't think we have documentation for README.md and monitoring.yml yet. Would it more prudent to use MAINTAINERS.md as an example?

Reply:
We do actually have an article for `monitoring.yml` :)

https://help.getsol.us/docs/packaging/monitoring.yml